src/app/components/accounts/activation-email/activation-email.component.ts

```diff
@@ -28,12 +29,12 @@ export class ActivationEmailComponent implements OnInit {
         });
 
         this.registerService.PostActivation(this.uuid, this.token)
-            .subscribe(response => {
+            .subscribe(() => {
                 this.messageService.add(MESSAGE_TYPES.SUCCESS, 'You have activated your account successfully.');
-                this.router.navigate(['/accounts/login']);
+                this.router.navigate(['/accounts/login']).then();
```

Comment:
I'm not sure what we should put in the `.then()` or if we just leave it empty. It throws a linting warning so I feel like we should include it.

Reply:
yeah `.then()` should be there.